docs/contributors/outreach.md

```diff
@@ -4,7 +4,7 @@ This includes articles, talks, demos and anything the community is doing to disc
 
 ## Articles
 
-A short list of useful articles around defining, extending, and contributing to Gutenberg.
+A short-list of useful articles around defining, extending, and contributing to Gutenberg.
```

Comment:
Don't think this is correct.

"This is a short list of [..]" is absolutely fine.

So, no hyphen.